public_html/wp-content/plugins/pattern-directory/includes/pattern-validation.php

```diff
@@ -375,3 +407,27 @@ function check_for_spam( $post ) {
 
 	return array( $is_spam, $spam_reason );
 }
+
+/**
+ * Helper function to check for a valid pattern title.
+ *
+ * @param string $title
+ * @return boolean
+ */
+function is_title_valid( $title ) {
+	// Check title against a list of disallowed words.
+	// Note the space after `test ` to avoid matching "testimonial".
+	$disallow_list = array( 'test ', 'testing', 'my pattern', 'wordpress', 'example' );
```

Comment:
Should we add `pattern` to the list?

Reply:
I though about that, but I think `pattern` could refer to a patterned background, like "Headline over a bright pattern"